packages/ui/core-components/src/lib/atoms/inputs/checkbox/Checkbox.svelte

```diff
@@ -27,29 +29,53 @@
 	export let hideDuringPrint = true;
 	$: hideDuringPrint = hideDuringPrint === 'true' || hideDuringPrint === true;
 
+	/** @type {boolean} */
+	export let checked = false;
+
+	/** @type {boolean | string} */
 	export let defaultValue = false;
 
-	$: $inputs[name] = toBoolean(defaultValue);
+	let isChecked = toBoolean(defaultValue) || checked;
```

Comment:
```suggestion
	let isChecked = toBoolean(defaultValue) || toBoolean(checked);
```

`checked` should use `toBoolean` as well if it comes from the user